src/cargo/ops/registry.rs

```diff
@@ -971,13 +971,21 @@ pub fn registry_logout(config: &Config, reg: Option<&str>) -> CargoResult<()> {
     Ok(())
 }
 
+pub enum SubCommand {
```

Comment:
At this layer of the abstraction, naming this "SubCommand" isn't clear.  In particular, this file is for a lot of different registry operations so its also not specific enough